pkg/providers/property/data_akamai_search.go

```diff
@@ -0,0 +1,115 @@
+package property
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/akamai/AkamaiOPEN-edgegrid-golang/v2/pkg/papi"
+	"github.com/akamai/terraform-provider-akamai/v2/pkg/akamai"
+	"github.com/akamai/terraform-provider-akamai/v2/pkg/tools"
+)
+
+func dataSourceSearch() *schema.Resource {
+	return &schema.Resource{
+		ReadContext: dataSourceSearchRead,
+		Schema: map[string]*schema.Schema{
+			"key": {
+				Type:             schema.TypeString,
+				Required:         true,
+				Description:      "Key must have one of three values: \"edgeHostname\", \"hostname\" or \"propertyName\"",
+				ValidateDiagFunc: tools.IsNotBlank,
+			},
+			"value": {
+				Type:             schema.TypeString,
+				Required:         true,
+				Description:      "Value of Search",
+				ValidateDiagFunc: tools.IsNotBlank,
+			},
+			"properties": {
+				Type:        schema.TypeList,
+				Computed:    true,
+				Description: "List of properties",
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"account_id":        {Type: schema.TypeString, Computed: true},
+						"asset_id":          {Type: schema.TypeString, Computed: true},
+						"contract_id":       {Type: schema.TypeString, Computed: true},
+						"group_id":          {Type: schema.TypeString, Computed: true},
+						"property_id":       {Type: schema.TypeString, Computed: true},
+						"property_version":  {Type: schema.TypeInt, Computed: true},
```

Comment:
please, execute `golangci-lint` on your PR

Reply:
I did using https://github.com/akamai/terraform-provider-akamai/blob/master/.golangci.yaml, is there something unexpected?

Reply:
I did just see some misaligned lines, I will double check now

Reply:
Oh, you were right - there were no linter issues 👍 

I apologize about the noise 😃